src/DGmethods/DGmodel.jl

```diff
@@ -398,6 +398,30 @@ function init_ode_state(
     return state
 end
 
+function restart_ode_state(
+    dg::DGModel,
+    s_state;
```

Comment:
What is `s_state` here?

Reply:
`s_state` is the `Q` that was saved. Strictly speaking, it is `Q.realdata`. I'll update the name to be clearer.

Reply:
Maybe include the type, too?